Scripts/Mac/InstallSharedDataQueueDylib.sh

```diff
@@ -0,0 +1,9 @@
+. "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
+
+BUILDDIR=$VFS_OUTPUTDIR/GVFS.Build
+cp $VFS_SRCDIR/nuget.config $BUILDDIR
+dotnet new classlib -n GVFS.Restore -o $BUILDDIR --force
+dotnet add $BUILDDIR/GVFS.Restore.csproj package --package-directory $VFS_PACKAGESDIR SharedDataQueueDylib --version '1.0.0'
```

Comment:
Just some naming nits:
* We're already using `GVFS.Restore.csproj` elsewhere, use a different name here to avoid confusion. And actually, it would be nice to rename both to be more descriptive to what they're actually restoring.
* "SharedDataQueueDyLib" is not very descriptive. I would give it a name that indicates that it's a temporary package with a bug fix.